components/debounce/common/pref_names.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/debounce/common/pref_names.h"
+
+#include "components/prefs/pref_registry_simple.h"
+
+namespace debounce {
+
+const char kDebouncePrefEnabled[] = "brave.debounce";
```

Comment:
`brave.debouce.enabled` and just `KDebounceEnabled`. The convention for namespace here is debounce::prefs which also makes it clear that it's a pref